frontend/pages/index.js

```diff
@@ -217,3 +219,15 @@ export default function Home() {
     </div>
   )
 }
+
+
+Index.getInitialProps = async function() {
+  const res = await fetch('http://localhost:3000/api/counties/[id].js');
+  const data = await res.json();
+  return {
+        data: data.data
```

Comment:
Maybe make this more specific because I think there will be additional props in the future.  

`selectedCountyData: data.data`